PanKanapka/Api.Web/Controllers/TasksController.cs

```diff
@@ -61,7 +67,7 @@ private WorkerTask GetTask()
                 {
                     new TaskItem()
                     {
-                        Date = "2009/10/05",
+                        Date = DateTime.Today,
```

Comment:
Please remove the whole GetTask method. We don't need it anymore.
Please use _tasksRepository in every method in TasksController.